### PR TITLE
Coverage and Test Parsing

### PR DESCRIPTION
**Pull request by cboltt**
- Adds three `awk` parsers:
    - `coverage.awk` parsers out the commit and the 4 types of coverage
    - `tests.awk` parsers out the commit, tests passed, and total tests
    - `format.awk` cleans up the parsed outputs (right now, it just removes an extra comma
- Restructures output files:
    - `/data/raw` contains all our raw data
    - `/data/` contains the final spreadsheets ready for visualization
    - I didn’t touch the changedTestFiles setup
- Adds example data files based on the latest script run in `/data/examples`

### CODE REVIEW

**Comment by cboltt** [comment]
Covers cases 1 and 2 from #4

**Comment by xavierdelacruz** [comment]
Hi @cbolton97 , 

The CSVS look great! It's just missing the header information, which I need in order to read data properly.

In this case, the first line of the coverage CSV would be

Commit,Line Coverage%,Statement Coverage%,Branch Coverage %,Function Coverage %

The same applies with the test:

Commit,Tests Passed,Tests

Thanks!

**Review by xavierdelacruz — APPROVED** [review]
The example csvs look good to me! exactly what the program is expecting

File: git-iterate [hunk 1]
```diff
@@ -26,16 +32,21 @@ git --git-dir=$1/.git worktree add $TEMP_DIR
 # get a list of all commits, omitting merges
 COMMITS=$(git -C $TEMP_DIR rev-list --no-merges --reverse HEAD) # add --max-count=<num>to limit commits to given number; for testing only. 
 
-:> $2 # clears the file before writing to it
-:> changedTestFiles.txt
+ # clears the file before writing to it
+> changedTestFiles.txt
```
Comment (lydiachoi) [inline git-iterate:36]: confirming we don't need the `:` in front of >

**Review by lydiachoi — APPROVED** [review]

**Pull request merged**